requirements.in

```diff
@@ -9,3 +9,4 @@ pytest-xdist==1.31.0
 pytest==5.3.5
 PyYAML==5.3
 sqlparse==0.3.0
+mozilla_schema_generator==0.1.4
```

Comment:
This was added as a dependency for finding the metadata related to the custom distributions.

Reply:
The library does use the probe info service, it just provides a nice API around it. The schema generator already walks the various endpoints to construct the list of probes from a list of dependencies.

https://github.com/mozilla/mozilla-schema-generator/blob/master/mozilla_schema_generator/glean_ping.py